### PR TITLE
feature #263: add default editor settings

### LINKED ISSUE #263: Feature request  : Add a way to define a default text editor for edit/edit-config commands
I suggest adding a new parameter in the settings to give to the user the liberty to choose his own editor as he which.

### PR DESCRIPTION
**Pull request by tdaudi**
resolve #263 

This feature adds the possibility to define a default text editor other than the default defined in the code.

### CODE REVIEW

**Comment by tdaudi** [comment]
Hello @sbstp, I took the liberty to make this tiny feature. What do you think ?

**Comment by sbstp** [comment]
It's a good idea, however I think setting the default_editor in the config should take precedence over the EDITOR environment variable.

File: README.md [hunk 1]
```diff
@@ -103,6 +103,11 @@ available below.
 # Default: unset
 shell: bash
 
+# For the commands `kubie edit/edit-config`
+# Possible values: Any installed text editor
+# Default: ""
```
Comment (sbstp) [inline README.md]: I think default should be "unset"

Comment (tdaudi) [reply README.md]: You are right, it is a documentation mistake I've done because it is actually unset.

**Comment by tdaudi** [comment]
This should be better :ok_hand:

**Comment by sbstp** [comment]
Thanks, I'll do a release soon

**Pull request merged**